charts/k8up/Chart.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
   - backup
   - operator
   - restic
-version: 4.6.0
+version: 4.7.0
```

Comment:
4.7.0 of the chart has already been released. (weird that this doesn't cause a merge conflict...)

Reply:
Indeed. I also rebased my branch to pickup the latest changes-